sql-statements/sql-statement-create-index.md

```diff
@@ -104,44 +104,108 @@ Query OK, 0 rows affected (0.31 sec)
 
 ## Expression index
 
+In some scenarios, query conditions run a filtering process based on a certain expression. In these scenarios, the query performance is relatively poor because ordinary indexes cannot take effect, the query can only be executed by scanning the entire table. The expression index is a type of special index that can be built on an expression. Once an expression index is built, TiDB can use the index for the expression-based query, which significantly improves the query performance.
```

Comment:
@en-jin19 Global comment: built index --> create index

Reply:
```suggestion
In some scenarios, query conditions run a filtering process based on a certain expression. In these scenarios, the query performance is relatively poor because ordinary indexes cannot take effect, the query can only be executed by scanning the entire table. The expression index is a type of special index that can be created on an expression. Once an expression index is created, TiDB can use the index for the expression-based query, which significantly improves the query performance.
```